mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -12,6 +12,7 @@
       </intent-filter>
 
     </activity>
+    <service android:name=".AppClearedService" android:stopWithTask="false" />
```

Comment:
I assume this is where you register the background service for Android? Did we have to do this for other background tasks?

Reply:
That's right. This registers the super tiny service that only has the job to listen on being killed by user (swipe up in recent apps) or system (out of memory etc). It's actually "running", i.e. sleeping all the time in the immich process.

The other "services" are not really services but rather tasks / pieces of work scheduled by the WorkManager (Android System). Thus, we don't need to register them (and actually we can't as they use another mechanism).